lib/transport_web/controllers/dataset_controller.ex

```diff
@@ -11,8 +11,8 @@ defmodule TransportWeb.DatasetController do
     |> render("index.html")
   end
 
-  def new(%Plug.Conn{} = conn, %{"slug" => slug}) do
-    case Organizations.get(conn, slug) do
+  def new(%Plug.Conn{} = conn, %{"id" => id}) do
```

Comment:
Same, this is our front's representation, it not the same context than our internal relationship with data.gouv.fr.